diffrax/_step_size_controller/adaptive.py

```diff
@@ -179,6 +179,52 @@ class PIDController(
         common to refer to solving an equation to specific tolerances, without
         necessarily stating which solver was used.)
 
+        !!! Example
```

Comment:
This is quite a long example, so let's maybe make it expandable with `???`

Reply:
That sounds great!